app/interactors/user/notify_datapass_of_token_ownership.rb

```diff
@@ -3,12 +3,16 @@ class NotifyDatapassOfTokenOwnership < ApplicationInteractor
     def call
       return unless first_login_after_token_transfer?
 
-      APIEntreprise::UserMailer.notify_datapass_for_data_reconciliation(context.user).deliver_later
+      mailer_klass.notify_datapass_for_data_reconciliation(context.user, context.api).deliver_later
       context.user.update(tokens_newly_transfered: false)
     end
 
     private
 
+    def mailer_klass
+      "API#{context.api.classify}::UserMailer".constantize
```

Comment:
Vis-à-vis de mon 2e commentaire (😅): passer directement namespace fonctionne.

```suggestion
      "#{context.namespace.classify}::UserMailer".constantize
```